source/reference/minio-mc/mc-mirror.rst

```diff
@@ -129,35 +118,27 @@ Parameters
 .. mc-cmd:: --encrypt-key
    
 
-   Encrypt or decrypt objects using server-side encryption with
-   client-specified keys. Specify key-value pairs as ``KEY=VALUE``.
+   Encrypt or decrypt objects using server-side encryption with client-specified keys. 
```

Comment:
So we don't have to answer this *now*, but it might be worth asking (@aead maybe?) whether this is a blind en/de crypt

How do we handle mixed encryption states? e.g. if the source has both encrypted and unencrypted objects (and for the sake of sanity lets assume that all encrypted objects use the same key), what happens on the destination? Do we decrypt the encrypted objects and encrypt the decrypted objects?

Reply:
Added https://github.com/minio/docs/issues/443 to cover this later.